system/integrations/cti/includes/recent-logs.include.rst

```diff
@@ -0,0 +1,27 @@
+Recent Logs
```

Comment:
"logs", not "Logs".

Reply:
This heading actually reflects the heading within the Zammad UI.

![image](https://user-images.githubusercontent.com/6549061/156400678-be87f3f3-bbf8-4657-8bf6-c7c870ae4a11.png)


Do you think that this is a bad idea?

Reply:
From my understanding it is "Recent logs", not "Recent Logs", as it is not a proper name. But to provide consistency with the Zammad UI, I support your suggestion to keep it as "Recent Logs".